modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,15 @@
+[id="authenticating-with-the-guest-user"]
+= Authenticating with the Guest user
+
+To start exploring {product} features, do not worry about authentication: after the default installation, you can authenticate with the Guest user, and use all the application features.
+
+.Prerequisites
+* You installed {product-short} by using default values.
+See link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}#Install%20and%20Upgrade[Installing RHDH].
```

Comment:
Does the user need to select the installation method they want to use? Or is there a general RHDH installation method, and not one specific to OCP, EKS, air-gapped, etc?

Reply:
Default install. Regardless of the platform. No need to introduce a non variable here.